services/movie.py

```diff
@@ -28,13 +32,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Using mutable default arguments like `list` is not recommended as it can lead to unexpected behavior. Consider using `None` as the default value and then initializing the list inside the function if needed.